app/views/visns/show.html.erb

```diff
@@ -4,7 +4,7 @@
   <%= javascript_include_tag '_practice_card_utilities', 'data-turbolinks-track': 'reload' %>
   <%= javascript_tag 'data-turbolinks-track': 'reload' do %>
     <%= render partial: 'show', formats: [:js] %>
-    <%# set visnNumber before loading the visns/show file %>
+    <%# set visnNumber before loading the crh/show file %>
```

Comment:
I think we want to change this back to `visn/show file`

Reply:
Yep.. i updated it.